star_tides/db/users.py

```diff
@@ -0,0 +1,51 @@
+'''star_tides.db.users
+
+This file contains classes and utilities for working with User objects in the
+KB database.
+'''
+
+from typing import List, NamedTuple, Text
+from enum import Enum
+
+
+class UserPrivelege(Enum):
+    UNKNOWN = 0
+    ADMIN = 1
+    COLLABORATOR = 2
+
+class User(NamedTuple):
+    id: Text
+    contact_id: Text
+    username: Text
+    login_email: Text
+    password_hash: Text
+    privilege: UserPrivelege
```

Comment:
This will actually be solved by this issue: https://github.com/STAR-TIDES/kb/issues/8. We will get the named tuple functionality in the interface level while also serving as the data model and mongo interaction point.